android/app/src/main/java/com/zulipmobile/notifications/NotifyReact.kt

```diff
@@ -4,12 +4,11 @@ package com.zulipmobile.notifications
 
 import android.os.Bundle
 import android.util.Log
-import com.facebook.react.ReactInstanceManager
-import com.facebook.react.ReactNativeHost
 import com.facebook.react.bridge.Arguments
 import com.facebook.react.bridge.ReactContext
-import com.facebook.react.common.LifecycleState
-import com.facebook.react.modules.core.DeviceEventManagerModule
```

Comment:
This line can probably get removed one commit sooner, at the same time a similar line appears in MainActivity.kt.